index.html

```diff
@@ -3,12 +3,15 @@
 <head>
 	<meta charset="UTF-8">
 	<link href="https://fonts.googleapis.com/css?family=BioRhyme:400&display=swap" rel="stylesheet">
+	<link href="https://fonts.googleapis.com/css2?family=Mali:wght@300&display=swap" rel="stylesheet"> 
+	<link href="https://fonts.googleapis.com/css2?family=Mali:wght@200&display=swap" rel="stylesheet"> 
 	<link rel="stylesheet" type="text/css" href="styles.css">
 	<script src="script.js" type="module"></script>
 	<title>Emoji Cards Game</title>
 </head>
 <body>
 	<aside id="landing-page">
+		<title-text>Emoji Card Game</title-text>
```

Comment:
title-text isn't a standard HTML element, could you change this to an `h1` and give that the id of "title-text"?